booking/src/main/java/org/dcsa/conformance/standards/booking/checks/BookingChecks.java

```diff
@@ -59,13 +59,18 @@ public class BookingChecks {
   private static final JsonPointer CARRIER_BOOKING_REQUEST_REFERENCE = JsonPointer.compile("/carrierBookingRequestReference");
   private static final JsonPointer CARRIER_BOOKING_REFERENCE = JsonPointer.compile("/carrierBookingReference");
   private static final JsonPointer BOOKING_STATUS = JsonPointer.compile("/bookingStatus");
-  public static ActionCheck requestContentChecks(UUID matched, Supplier<CarrierScenarioParameters> cspSupplier, Supplier<DynamicScenarioParameters> dspSupplier) {
+  public static ActionCheck requestContentChecks(UUID matched, String standardsVersion, Supplier<CarrierScenarioParameters> cspSupplier, Supplier<DynamicScenarioParameters> dspSupplier) {
     var checks = new ArrayList<>(STATIC_BOOKING_CHECKS);
+    if (standardsVersion.equals("2.0.0-Beta-1")) {
+      checks.add(IS_EXPORT_DECLARATION_REFERENCE_PRESENT);
+      checks.add(IS_IMPORT_DECLARATION_REFERENCE_PRESENT);
+    }
```

Comment:
In DT-938, Henrik mentions an inverse check (`isExportDeclarationRequired = false` must imply that `exportDeclarationReference` is absent). I do not see that added here (nor in the later place where similar `if` is present).

What happened with this check? Did you agree to omit that (without updating the issue) or was the omission unintentional?

Reply:
There was no agreement on this topic, as we were suppose to discuss this in the Chapter days and hence i didnt include this check. But will check with @HenrikHL  whether this can be included or not.

Reply:
Hi @nt-gt Niels, @HenrikHL and me agreed that we will add this check as of now for isExportDeclarationRequired and isImportLicenseRequired field. Anyhow we have to discuss this point with @XVsTech in order to streamline it.